infrastructure/templates/infrastructure/powerplant_detail.html

```diff
@@ -132,6 +145,23 @@ <h3>CO2 Emissions</h3>
       </div>
     </section>
 
+    <section class="related-power-data">
+        <h2>Associated Project Initiatives</h2>
+        
+        {% for project in object.project_set.published %}
+        <div class="row">
+            <section>
+                <h4>{{ project.name }}</h4>
```

Comment:
This prints all related project names even if there are no associated initiatives.

<img width="851" alt="screen shot 2018-12-28 at 5 17 44 pm" src="https://user-images.githubusercontent.com/129679/50529661-93242c80-0ac4-11e9-978e-5d6d48405be3.png">

Reply:
Should handle this.
```
{% if project.initiatives.all %}
```